projects/vdk-plugins/vdk-logging-json/src/vdk/plugin/logging_json/logging_json.py

```diff
@@ -46,25 +50,38 @@ def format(self, record):
         return json.dumps(result)
 
 
+@hookimpl
+def vdk_configure(config_builder: ConfigurationBuilder):
+    config_builder.add(
+        key=LOGGING_FORMAT,
+        default_value="TEXT",
+        description="The format in which to structure VDK logs.",
```

Comment:
What are the possible values ?  Basically it's a feature flag (disable /enable json formatting). 


Outside of scope of this PR: 

Also, it looks strange for vdk-logging-json plugin to have a format. I wonder if we should have one plugin - logging-format with options ??

Reply:
created one plugin for both formats, deleted other plugins